fleet/view/js/fleet.js

```diff
@@ -0,0 +1,316 @@
+var loc = window.location.pathname;
+var split_url = loc.split('/');
+var url_host = window.location.origin+"/"+split_url[1]+"/"+split_url[2];
+
+var loc_host = window.location.origin
+var split_host = loc_host.split('/');
+
+var id = 0;
+
+if (split_host[2] != 'localhost') {
+    if (window.location.pathname.split('/')[3]) {
+        id = window.location.pathname.split('/')[3];
+    }
+} else {
+    if (window.location.pathname.split('/')[4]) {
+        id = window.location.pathname.split('/')[4];
+    }
+}
+var counter_doc_type = 0;
+
+document_type = []
+
+function typeDocument() {
+    $.ajax({
+          type: "get",
+          url: url_path + "/fleet?para=type_doc",
+          data : {}
+    }).done(function(data) {
+		if (data['data'].length > 0) {
+			document_type = data['data'];
+		}
+    });
+}
+
+$(document).ready(function() {
+
+    typeDocument();
+    
+    var doc_list = $("#doc_list").DataTable({
+        "paging": false,
+        "ordering": false,
+        "info": false,
+        "searching": false,
+        "language": {
+            "emptyTable": "No include item"
+        },
+        autoWidth : false,
+        bAutoWidth: false, 
+        fixedHeader : {
+          header: true, 
+        },
+        scrollY: "120px",
+        "scrollX": true,
+        aoColumns: [
+            { sWidth: '2%' },
+            { sWidth: '30%' },
+            { sWidth: '20%' },
+            { sWidth: '25%' },
+            { sWidth: '23%' },
+        ],
+        
+    });
+    
+    var counter = 1;
+
+    $('#add_doc').on('click', function() {
+        if (counter > document_type.length) {
+            alert('List Document Exceeded');
+        } else {
+            var sudah_dipilih = []
+            $('#doc_list tbody tr').each(function(){
+                if ($.parseHTML($(this).find('td:eq(1)').find('select').val())){
+                    sudah_dipilih.push(parseInt($.parseHTML($(this).find('td:eq(1)').find('select').val())[0]['data']));
+                }
+            });
+            var select_type = '';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/davidgunadi/hello-world/pullRequest?prid=8400024)
```suggestion
            var select_type = "";
```